Fix remove unnecessary overview headings (628) (#632)
astronomer/docs: software/integrate-iam.md

description: Append IAM roles to an Airflow Deployment on Astronomer Software.
---

> On Astronomer, IAM roles can be appended to the Webserver, Scheduler and Worker pods within any individual Airflow Deployment on the platform.

Reviewer: ```suggestion
On Astronomer, IAM roles can be appended to the webserver, scheduler and worker pods within any individual Airflow Deployment on the platform.
```

IAM roles on [AWS](https://aws.amazon.com/iam/faqs/) and other platforms are often used to manage the level of access a specific user (or object, or group of users) has to some resource (or set of resources). The resource in question could be an S3 bucket or Secret Backend, both of which are commonly used in tandem with Airflow and Astronomer and can now be configured to be accessible only to a subset of Kubernetes pods within your wider Astronomer cluster.

> ## Limitations

Reviewer 2: ```suggestion
## Implementation Considerations
```
I don't think all of these are limitations

> The following are the known limitations when you integrate IAM roles:

Reviewer 2: ```suggestion
Consider the following when you integrate IAM roles:
```

> * All pods within your Airflow Deployment assume the IAM role. There is currently no way to use more than 1 IAM role per deployment.

Reviewer: ```suggestion
* All pods within your Airflow Deployment assume the IAM role. There is currently no way to use more than one IAM role per Deployment.
```

> * If you’d like your IAM role to apply to more than 1 deployment, you must annotate each deployment.

Reviewer: ```suggestion
* If you’d like your IAM role to apply to more than one Deployment, you must annotate each Deployment.
```
